static/app/components/structuredEventData/index.tsx

```diff
@@ -324,10 +329,12 @@ export default function StructuredEventData({
   data = null,
   withAnnotatedText = false,
   forceDefaultExpand,
+  showCopyButton,
+  onCopy,
   ...props
 }: StructuredEventDataProps) {
   return (
-    <pre {...props}>
+    <pre {...props} style={{position: 'relative'}}>
```

Comment:
One last thing - can we add these styles with CSS in a styled component instead?

Reply:
yup!